packages/client/src/modules/leaderBoardModule/PlayerItem.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC } from 'react'
+import styled from 'styled-components'
+import { P1 } from '../../assets/styles/texts'
+import defaultAvatar from '../../assets/images/catAvatar.png'
+import { media } from '../../assets/styles/media'
+
+type Props = {
+  avatarUrl?: string|null
+  name: string
+  rating: number
+  points: number
+}
```

Comment:
Можешь переместить `PlayerItemType` сюда, главное чтобы он был и ты бы его переиспользовал в нужных местах

```suggestion
type Props = Omit<PlayerItemType, 'id'>
```